code/modules/reagents/Chemistry-Recipes/Pyrotechnics.dm

```diff
@@ -18,13 +57,6 @@
 	required_reagents = list("iron" = 1, "oxygen" = 1, "hydrogen" = 1)
 	result_amount = 3
 
-/datum/reagent/clf3
-	name = "Chlorine Trifluoride"
-	id = "clf3"
-	description = "Makes a temporary 3x3 fireball when it comes into existence, so be careful when mixing. ClF3 applied to a surface burns things that wouldn't otherwise burn, sometimes through the very floors of the station and exposing it to the vacuum of space."
-	reagent_state = LIQUID
-	color = "#FF0000"
-	metabolization_rate = 4
 
 /datum/chemical_reaction/clf3
```

Comment:
Because the original chemicals already separated them, plus it makes smaller files.